### PR TITLE
Pptp 2387

### PR DESCRIPTION
**Pull request by mauro-fanton**
### Description of Work carried through

Brief description of what/why/how.

### CODE REVIEW

File: a11y/views/ExportedPlasticPackagingWeightAccSpec.scala [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+/*
+ * Copyright 2022 HM Revenue & Customs
```
Comment (liampan) [inline a11y/views/ExportedPlasticPackagingWeightAccSpec.scala:20]: two lots of copy write 🤔

File: a11y/views/ExportedPlasticPackagingWeightAccSpec.scala [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    with AccessibilityMatchers {
+
+  val form = new ExportedPlasticPackagingWeightFormProvider()()
+
+  val page = inject[ExportedPlasticPackagingWeightView]
+
+  private def createView: Html =
+    page(form, NormalMode, 1234L)(request, messages)
+
+  "returns.ExportedPlasticPackagingWeightAccSpec" should {
```
Comment (liampan) [inline a11y/views/ExportedPlasticPackagingWeightAccSpec.scala:53]: this should be the name of the view not the spec

**Review by liampan — APPROVED** [review]
Approving, just a few little coments to consider before merging, maybe we do them in a separate pr though :)

File: a11y/views/ExportedPlasticPackagingWeightAccSpec.scala (hunk 1 above)
Comment (mauro-fanton) [reply a11y/views/ExportedPlasticPackagingWeightAccSpec.scala]: sorted it. Thanks

File: a11y/views/ExportedPlasticPackagingWeightAccSpec.scala (hunk 2 above)
Comment (mauro-fanton) [reply a11y/views/ExportedPlasticPackagingWeightAccSpec.scala]: yes, agree. done

File: a11y/views/ExportedPlasticPackagingWeightAccSpec.scala [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+ */
+
+import base.ViewSpecBase
+import forms.returns.ExportedPlasticPackagingWeightFormProvider
+import models.Mode.NormalMode
+import play.twirl.api.Html
+import uk.gov.hmrc.scalatestaccessibilitylinter.AccessibilityMatchers
+import views.html.returns.ExportedPlasticPackagingWeightView
+
+class ExportedPlasticPackagingWeightAccSpec
```
[the remark this answers is not in the document]
Comment (mauro-fanton) [reply a11y/views/ExportedPlasticPackagingWeightAccSpec.scala:26]: yes, we can look at this when we start add more test. Thanks

**Review by PSouthern — CHANGES_REQUESTED** [review]
Can you add something to the readme explaining how to run the tests please?

**Pull request merged**